RULES_AND_GUIDELINES.md

```diff
@@ -0,0 +1,220 @@
+
+[^ back to top](#)
+
+## <a name="limited common-property"></a>Limited Common Property
+
+Limited common property is defined as spaces commonly owned by the association, but designated for use by an individual or limited group of owners. Every unit in SC3 is allotted a limited common space, with a perimeter bound by the original construction of the outside fence and the exterior surface of the building, designated as a ‘fenced yard.’ Owners and residents have discretionary use of the fenced yard designated as the limited common element for the given unit. However, within the fenced yard, no items may be invasively attached to the exterior of the building (no nails or screws) and no siding may be cut or damaged. Owners are free to change landscaping within their fenced yard but may not change, adjust or modify any structural retaining walls that are present for the common good. Man-made features may be no taller than 70” total or 24” taller than the approved 46” fence. Man-made features can only be attached to the fence at the owner's discretion. All landscaping, repairs, maintenance and costs associated with the limited common space is the sole responsibility of the owner. Any item within a fenced yard that is in violation of common elements (e.g., shelf screwed to the wall, nails in siding, pet door through wall) may be removed without notice at the board's discretion and all associated expenses will be the owner's sole responsibility.
```

Comment:
There's has to be a way where nails and screws are fine. Everyone's cable connection is fastened to the exteriors of the walls with fasteners and nails/screws.